### ISSUE TITLE
Album rotation - multiple images

### ISSUE THREAD

**Issue by NullVsNone** [tlstommy/PiInk#6]
Could this be enhanced to support uploading multiple images, with a set timer to rotate between the "album" of images?

Not very experienced in python (nor Pi(s)), but something along the lines of:

- No longer calling delete image

https://github.com/tlstommy/PiInk/blob/511791410bd8a3af3004ea20ad743f785f81900f/src/webserver.py#L89

- [Optional] Use a UUID for filename for multiple uploads

https://github.com/tlstommy/PiInk/blob/511791410bd8a3af3004ea20ad743f785f81900f/src/webserver.py#L90

- Introducing an updater

```python
import sched, time, random

def cycle_album(scheduler): 
    scheduler.enter(3600, 1, cycle_album, (scheduler,))
    updateEink(random.choice(UPLOAD_FOLDER, ORIENTATION,ADJUST_AR)

album_schedule = sched.scheduler(time.time, time.sleep)
album_schedule.enter(3600, 1, cycle_album, (album_schedule,))
album_schedule.run()
```

**Comment by tlstommy** [comment]
Yes! This is actually something I've been planning on implementing once I have more time!

Thanks for your suggestions!

**Comment by nizh0001** [comment]
Hey @tlstommy and @NullVsNone,

We’re a student team working on a free mobile app to make it easier for digital picture frame users to crop, upload, and manage their photos. We really want to build something useful for the community, and your discussion about rotating albums and handling multiple images is super relevant to what we're exploring.

We’ve made a quick anonymous survey (5–10 minutes) to better understand how people actually use digital frames. If you have a moment, we’d really appreciate your input:

👉 https://forms.office.com/Pages/ResponsePage.aspx?id=JNkb7GoKqUqqicmAMWwESROZ7FgoNnZNtKAU0uwc2b9URjJaQUZPV1VXWUtCQllBRUZKT1lXT1dQMC4u

Thanks a lot for helping us out!
— TechnoLab Student Team
